app/javascript/packs/new_timelog.jsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react'
+import ReactDOM from 'react-dom'
+import PropTypes from 'prop-types'
+import Select from 'react-normalized-select'
+
+class NewTimelog extends React.Component {
+
+  constructor(props) {
+    super(props);
+    this.state = { card: false };
+  }
+
+  handleClick() {
+    var start_time  = this.refs.start_time.value;
```

Comment:
Look for `const` and `let` in ES6. Consider in future to use `let` where was `var` and `const` for all immutable variables.